src/modules/ROOT/pages/changelog/202303/01.json

```diff
@@ -0,0 +1,889 @@
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/877/commits/cfc4ab7424ed1a131bb34cfaed8c4ca18eef1b07"
+      ]
+    },
+    {
+      "what": [
+        "support for conditional fields.",
+        "Fields in records can now be displayed based on evaluated expressions."
+      ],
+      "why": [
+        "to give users more control on how records are displayed."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/770/commits/1d286aff83f012e33cabc3456ffa175f7c105eb7"
+      ]
+    },
+    {
+      "what": [
+        "support to allow users to navigate to previous and next record page.",
+        "Added a checkbox to enable the record page pagination on the configurator record list."
+      ],
+      "why": [
+        "to allow users to navigate to previous and next record page."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/836/commits/ce79e8233d40894657c00e806fa50589c2724c92"
+      ]
+    },
+    {
+      "what": [
+        "support for bulk record updates."
+      ],
+      "why": [
+        "to allow the users to update multiple records in a record list."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/906/commits/97a9cd877ab11917dac9f031db36cbd8f4d215b3",
+        "https://github.com/cortezaproject/corteza/pull/906/commits/717e95b22efefc329198dd4689b0321273813cf8",
+        "https://github.com/cortezaproject/corteza/pull/906/commits/f8a3da0bdf40e010fde9902cc2954a6d5973b75d",
+        "https://github.com/cortezaproject/corteza/pull/906/commits/656bea5636f690cdf2697349695011734f24bf49",
+        "https://github.com/cortezaproject/corteza/pull/906/commits/508e8bab92d6eb2ca6fce17c807d4a33d79ae9ad"
+      ]
+    },
+    {
+      "what": [
+        "support for tabs page block in compose."
+      ],
+      "why": [
+        "to make page block organizing much easier for users."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/670/commits/c6f88e2679efe78e5319665ea754c19d7b2ed138",
+        "https://github.com/cortezaproject/corteza/pull/670/commits/a4308e8d88cdda2f13e3f19231b7815d6812d79e"
+      ]
+    },
+    {
+      "what": [
+        "support for managing and generating user's profile picture."
+      ],
+      "why": [
+        "to provide users with greater flexibility and customization options for their profile avatars, while also allowing admins to manage avatars on behalf of users."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/887/commits/e24b391e5b1dc539855122534a4394a6868c4afd"
+      ]
+    },
+    {
+      "what": [
+        "support for record list filters.",
+        "Record list filters can be configured on the record list configurator.",
+        "They can be grouped by roles and can be quickly selected from the filter dropdowns."
```

Comment:
reword this to point out what exactly was added.
Record list filters were there for a while now.
What exactly changed? Point that out